index.html

```diff
@@ -33,87 +39,99 @@ <h1 class="playfair dark-blue">Zilola Nazarova</h1>
-        </ul>
-        <a href="#" class="see-project white poppins">See this project</a>
+          <ul class="category-list dark-blue poppins">
+            <li class="category">HTML/CSS</li>
+            <li class="category">Ruby on Rails</li>
+            <li class="category">JavaScript</li>
+          </ul>
+          <a href="#" class="see-project white poppins">See this project</a>
         </div>
       </div>
-      <div class="work-card">
-        <img class="project-preview" src="./images/project-4.png" alt="Preview of the project-1">
+      <div class="work-card w4">
+        <img class="project-preview" src="./images/project-4.png" alt="Preview of the project-4">
+        <img class="project-preview" src="./images/project-4-big.png" alt="Preview of the project-4">
         <div class="work-info">
           <p class="project-name white poppins">Project name goes here</p>
-        <ul class="category-list dark-blue poppins">
-          <li class="category">HTML/CSS</li>
-          <li class="category">Ruby on Rails</li>
-          <li class="category">JavaScript</li>
-        </ul>
-        <a href="#" class="see-project white poppins">See this project</a>
+          <ul class="category-list dark-blue poppins">
+            <li class="category">HTML/CSS</li>
+            <li class="category">Ruby on Rails</li>
+            <li class="category">JavaScript</li>
+          </ul>
+          <a href="#" class="see-project white poppins">See this project</a>
         </div>
       </div>
-      <div class="work-card">
-        <img class="project-preview" src="./images/project-5.png" alt="Preview of the project-1">
+      <div class="work-card w5">
+        <img class="project-preview" src="./images/project-5.png" alt="Preview of the project-5">
+        <img class="project-preview" src="./images/project-5-big.png" alt="Preview of the project-5">
         <div class="work-info">
           <p class="project-name white poppins">Project name goes here</p>
-        <ul class="category-list dark-blue poppins">
-          <li class="category">HTML/CSS</li>
-          <li class="category">Ruby on Rails</li>
-          <li class="category">JavaScript</li>
-        </ul>
-        <a href="#" class="see-project white poppins">See this project</a>
+          <ul class="category-list dark-blue poppins">
+            <li class="category">HTML/CSS</li>
+            <li class="category">Ruby on Rails</li>
+            <li class="category">JavaScript</li>
+          </ul>
+          <a href="#" class="see-project white poppins">See this project</a>
         </div>
       </div>
-      <div class="work-card">
-        <img class="project-preview" src="./images/project-6.png" alt="Preview of the project-1">
+      <div class="work-card w6">
+        <img class="project-preview" src="./images/project-6.png" alt="Preview of the project-6">
+        <img class="project-preview" src="./images/project-6-big.png" alt="Preview of the project-6">
```

Comment:
Please make sure you used the correct image on your `index.html` file as it is in the `images` folder. Currently the `Project-6-big.png` is not displayed on the browser since you used a small `p` instead of a capital `P` when you call the image. Kindly fix it.